conf/application.code.conf

```diff
@@ -1,7 +0,0 @@
-include "application"
```

Comment:
I removed this AFAIK this is not being called in CODE and `application.conf` is being used.